.github/workflows/main.yml

```diff
@@ -390,36 +390,45 @@ jobs:
           $release_str = Get-Date -Format "yyyyMMdd"
           .\StoreBroker\Extensions\ConvertFrom-ExistingSubmission.ps1 -AppId $APP_ID -Release $release_str -OutPath "MSAppPayload/PDPs"
 
-          # Wipe out the Release string in PDP.xml to avoid the release version folder
-          [xml]$xmlDoc = Get-Content "MSAppPayload/PDPs/en-us/PDP.xml"
-          $xmlDoc.ProductDescription.Release = ""
-          $xmlDoc.Save( "MSAppPayload/PDPs/en-us/PDP.xml" )
-
-          echo "Going to create dummy screenshots for Application Payload creation:"
-          $xmlDoc.ProductDescription.ScreenshotCaptions.Caption | ForEach-Object {
-            $dummy_screenshot = "MSAppPayload/PDPs/en-us/" + $_.DesktopImage
-            echo $dummy_screenshot
-            echo NULL > $dummy_screenshot
-          }
-
-          echo "Going to create dummy additional assets for Application Payload creation:"
-          $xmlDoc.ProductDescription.AdditionalAssets.ChildNodes.FileName | ForEach-Object {
-            if ( -not $_ ) { return }
-            $dummy_assest = "MSAppPayload/PDPs/en-us/" + $_
-            echo $dummy_assest
-            echo NULL > $dummy_assest
-          }
+          # Deal every locale's PDP.xml. For example, "en-us/PDP.xml", "es-419/PDP.xml" ...
+          Get-Childitem -Path "MSAppPayload\PDPs\" -Include *.xml -File -Recurse | ForEach-Object {
+            $pdp = $_
+            $pdp_folder = Split-Path -parent $pdp
+
+            # Wipe out the Release string in PDP.xml to avoid the release version folder
+            [xml]$xmlDoc = Get-Content $pdp
+            $xmlDoc.ProductDescription.Release = ""
+            $xmlDoc.Save( $pdp )
+
+            echo "Going to create dummy screenshots for Application Payload creation:"
+            $xmlDoc.ProductDescription.ScreenshotCaptions.Caption | ForEach-Object {
+              if ( -not $_ ) { return }
+              $dummy_screenshot = "$pdp_folder/" + $_.DesktopImage
+              echo $dummy_screenshot
+              echo NULL > $dummy_screenshot
+            }
 
-          echo "Going to create dummy trailers & their images for Application Payload creation:"
-          $xmlDoc.ProductDescription.Trailers.Trailer | ForEach-Object {
-            $dummy_trailer = "MSAppPayload/PDPs/en-us/" + $_.FileName
-            echo $dummy_trailer
-            echo NULL > $dummy_trailer
+            echo "Going to create dummy additional assets for Application Payload creation:"
+            $xmlDoc.ProductDescription.AdditionalAssets.ChildNodes.FileName | ForEach-Object {
+              if ( -not $_ ) { return }
+              $dummy_assest = "$pdp_folder/" + $_
```

Comment:
It's a preëxisting problem but “assest” should be “asset” in this variable name.

Reply:
Good catch!  Thanks